{"uid":"8137e49223428161","name":"test_scale_down","fullName":"tests.integration.test_subordinates#test_scale_down","historyId":"d49a641025202176422160b3d71ecba3","time":{"start":1713150665592,"stop":1713150665592,"duration":0},"status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 340, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 240, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 85, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 843, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 832, in _runtest_for\n    yield\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 82, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 158, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 515, in setup\n    raise exc\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 512, in setup\n    col.setup()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1775, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 687, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 541, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 570, in _get_active_fixturedef\n    self._compute_fixture_value(fixturedef)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 647, in _compute_fixture_value\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1079, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 223, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 191, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop","time":{"start":1713149911174,"stop":1713149911200,"duration":26},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1713149911200,"stop":1713149911200,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"github_secrets","time":{"start":1713149911200,"stop":1713149911200,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1713149911201,"stop":1713149911339,"duration":138},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1713150665593,"stop":1713150665593,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 223, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 191, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 340, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 240, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 85, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 843, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 832, in _runtest_for\n    yield\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 82, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 158, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 515, in setup\n    raise exc\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 512, in setup\n    col.setup()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1775, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 687, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 541, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 570, in _get_active_fixturedef\n    self._compute_fixture_value(fixturedef)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 647, in _compute_fixture_value\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1079, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 223, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 191, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","steps":[],"attachments":[{"uid":"147aef2370318947","name":"log","source":"147aef2370318947.txt","type":"text/plain","size":6646}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"ops_test::finalizer","time":{"start":1713150665598,"stop":1713150665969,"duration":371},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1713150665970,"stop":1713150665970,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1713150665970,"stop":1713150665970,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1713150665970,"stop":1713150665970,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_subordinates"},{"name":"host","value":"fv-az984-579"},{"name":"thread","value":"8484-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_subordinates"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":2,"passed":37,"unknown":0,"total":39},"items":[{"uid":"f23a500b871e734b","reportUrl":"../2447//#testresult/f23a500b871e734b","status":"passed","time":{"start":1713059371220,"stop":1713059473365,"duration":102145}},{"uid":"f48babcc120ce73e","reportUrl":"../2445//#testresult/f48babcc120ce73e","status":"passed","time":{"start":1712972699764,"stop":1712972795715,"duration":95951}},{"uid":"a7b6447a8139bded","reportUrl":"../2430//#testresult/a7b6447a8139bded","status":"passed","time":{"start":1712886615970,"stop":1712886696753,"duration":80783}},{"uid":"c6c73cd3816cef38","reportUrl":"../2424//#testresult/c6c73cd3816cef38","status":"passed","time":{"start":1712800299736,"stop":1712800412328,"duration":112592}},{"uid":"63b90362b8c76608","reportUrl":"../2419//#testresult/63b90362b8c76608","status":"passed","time":{"start":1712713894111,"stop":1712713992600,"duration":98489}},{"uid":"8fd0080c6fa15c80","reportUrl":"../2416//#testresult/8fd0080c6fa15c80","status":"passed","time":{"start":1712627540943,"stop":1712627642881,"duration":101938}},{"uid":"54560e310a410e05","reportUrl":"../2409//#testresult/54560e310a410e05","status":"passed","time":{"start":1712541024356,"stop":1712541108644,"duration":84288}},{"uid":"5b5f75e2ff721412","reportUrl":"../2408//#testresult/5b5f75e2ff721412","status":"passed","time":{"start":1712454694969,"stop":1712454793591,"duration":98622}},{"uid":"937c92e0a76360d8","reportUrl":"../2407//#testresult/937c92e0a76360d8","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1712367295487,"stop":1712367295487,"duration":0}},{"uid":"b08b3d4ff74b59c0","reportUrl":"../2362//#testresult/b08b3d4ff74b59c0","status":"passed","time":{"start":1712281788812,"stop":1712281890690,"duration":101878}},{"uid":"3a454422ed4d9d73","reportUrl":"../2343//#testresult/3a454422ed4d9d73","status":"passed","time":{"start":1712195358534,"stop":1712195442583,"duration":84049}},{"uid":"eb9364dfc8dcdb6f","reportUrl":"../2330//#testresult/eb9364dfc8dcdb6f","status":"passed","time":{"start":1712109232029,"stop":1712109337731,"duration":105702}},{"uid":"22f8e89cc24bf55d","reportUrl":"../2327//#testresult/22f8e89cc24bf55d","status":"passed","time":{"start":1712022639109,"stop":1712022719136,"duration":80027}},{"uid":"b8082c26107b4f6f","reportUrl":"../2324//#testresult/b8082c26107b4f6f","status":"passed","time":{"start":1711936213539,"stop":1711936311552,"duration":98013}},{"uid":"ea1675e5dfc9a415","reportUrl":"../2323//#testresult/ea1675e5dfc9a415","status":"passed","time":{"start":1711849932067,"stop":1711850016188,"duration":84121}},{"uid":"e36c960a2dfa6119","reportUrl":"../2322//#testresult/e36c960a2dfa6119","status":"passed","time":{"start":1711763313065,"stop":1711763426626,"duration":113561}},{"uid":"d903e706671b89fe","reportUrl":"../2318//#testresult/d903e706671b89fe","status":"passed","time":{"start":1711676968149,"stop":1711677043393,"duration":75244}},{"uid":"a73eb11c89ecd92e","reportUrl":"../2315//#testresult/a73eb11c89ecd92e","status":"passed","time":{"start":1711590634058,"stop":1711590752010,"duration":117952}},{"uid":"3ebebb27e08a6742","reportUrl":"../2313//#testresult/3ebebb27e08a6742","status":"passed","time":{"start":1711504013854,"stop":1711504112644,"duration":98790}},{"uid":"1a036a2fc8a6a551","reportUrl":"../2309//#testresult/1a036a2fc8a6a551","status":"passed","time":{"start":1711417758768,"stop":1711417873570,"duration":114802}}]},"tags":["asyncio","group(1)"]},"source":"8137e49223428161.json","parameterValues":[]}